client.go

```diff
@@ -18,8 +18,17 @@ func (p *Provider) NewSession() error {
 	if p.MaxRetries == 0 {
 		p.MaxRetries = 5
 	}
-	sess, err := session.NewSession(&aws.Config{
-		MaxRetries: aws.Int(p.MaxRetries),
+	sess, err := session.NewSessionWithOptions(session.Options{
+		// Specify profile to load for the session's config
+		Profile: p.AwsProfile,
+
+		// Provide SDK Config options, such as Region.
```

Comment:
Region? Where?

Reply:
Hi, I think we can leave out aws-region.. Route53 doesnt require region since it is available globally. 

![Screen Shot 2020-11-24 at 7 27 27 AM](https://user-images.githubusercontent.com/9730242/100026852-83cfca00-2e26-11eb-958c-9b85aa37b5f6.png)

---

eitherway, should there be a need for us to specify a region (which is unlikely), it should be done in the `~/.aws/config` file, not directly on the code.

Reply:
Same thing here, please remove the comment.